pages/airdrops/create.tsx

```diff
@@ -289,209 +321,235 @@ const CreateAirdrop: NextPage = () => {
 
       <div className="grid grid-cols-2 gap-8">
         {/* project name */}
-        <div className="flex flex-col space-y-2">
-          <label className="flex items-center space-x-2">
-            <span className="font-bold">Project Name</span>
-            <TooltipIcon label="Enter your new airdrop name" />
-          </label>
-          <input
-            type="text"
-            className={clsx(
-              'bg-white/10 rounded border-2 border-white/20 form-input',
-              'placeholder:text-white/50',
-              'focus:ring focus:ring-plumbus-20'
-            )}
+        <FormControl
+          title="Name"
+          subtitle="This is how people will find you in the list of airdrops."
+          htmlId="airdrop-name"
+        >
+          <Input
+            id="airdrop-name"
+            name="name"
             placeholder="My Awesome Airdrop"
             value={projectName}
             onChange={(e) => setProjectName(e.target.value)}
           />
-        </div>
+        </FormControl>
 
         {/* CW20 token address */}
-        <div className="flex flex-col space-y-2">
-          <label className="flex items-center space-x-2">
-            <span className="font-bold">CW20 Token Address</span>
-            <TooltipIcon label="Enter your CW20 token address" />
-          </label>
-          <input
-            type="text"
-            className={clsx(
-              'bg-white/10 rounded border-2 border-white/20 form-input',
-              'placeholder:text-white/50',
-              'focus:ring focus:ring-plumbus-20'
-            )}
+        <FormControl
+          title="CW20 Address"
+          subtitle=" Address of the CW20 token that will be airdropped."
+          htmlId="airdrop-cw20"
+        >
+          <Input
+            id="airdrop-cw20"
+            name="cw20"
+            placeholder="juno1234567890abcdefghijklmnopqrstuvwxyz..."
             value={cw20TokenAddress}
             onChange={(e) => setCW20TokenAddress(e.target.value)}
```

Comment:
Regardless of library used, I'm curious if anyone has a plan for a validation rule to check if the string is a valid wallet address 👀

Reply:
Probably in rhf we can add a custom validator whether the length is correct and it has a `juno` prefix. Don't know if this should be in this PR scope or not.